packages/template-retail-react-app/app/hooks/use-password-reset.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {AuthHelpers, useAuthHelper} from '@salesforce/commerce-sdk-react'
+import {useToast} from '@salesforce/retail-react-app/app/components/shared/ui'
```

Comment:
We have a custom hook for toast which handles duplicated toast. `app/hooks/use-toast.js`